pkg/cluster/cluster_config.go

```diff
@@ -50,19 +50,27 @@ func IsNotReservedNamespace(ns *corev1.Namespace) bool {
 		ns.GetName() != "default" && ns.GetName() != "openshift"
 }
 
-// GetClusterServiceVersion retries the clusterserviceversions available in the operator namespace.
-func GetClusterServiceVersion(ctx context.Context, c client.Client, watchNameSpace string) (*ofapiv1alpha1.ClusterServiceVersion, error) {
+// GetClusterServiceVersion retries CSV only from the defined namespace with 100 as chucksize.
```

Comment:
I am not quite sure it's important to reveal implementation detail here. And there's a little typo too :)

```suggestion
// GetClusterServiceVersion retries CSV only from the defined namespace.
```

![chuck-size](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExOGtwaDdiaGlxdDAyc3lhNXE4bDVkMm9xZmI2OTBmM2t6NnZtZTM0byZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/7qZ3ZX1Gu3TZm/giphy.gif)

Reply:
lesson learnt today: i have been misspelling chunk in my (past)life.....</3